templates/deployment.yaml

```
@@ -37,6 +37,7 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       serviceAccountName: {{ include "benthos.serviceAccountName" . }}
+      restartPolicy: {{ .Values.podRestartPolicy }}
```

```suggestion
      restartPolicy: {{ .Values.deployment.restartPolicy }}
```
I think this would make more sense under the `deployment` property in the values